Libraries/SDHC/ff13/Source/diskio.h

```diff
@@ -14,9 +14,11 @@ extern "C" {
 #ifdef NATIVE_SDHC
 #include "sdhc_lib.h"
 #endif
-
+#ifdef FLASH
+#include "mscmem.h"
+#endif
 #include "rtc.h"
-
+#include "integer.h"
```

Comment:
`diskio.h` modification - Including `integer.h` should be done in the implementation file if necessary